sipXyealink/etc/yealink/types.xml

```diff
@@ -200,7 +201,7 @@
 </type>
 <type id="InfoType_type">
     <enum>
-	<option><label>Disabled</label><value>0</value></option>
+	<!--<option><label>Disabled</label><value>0</value></option>-->
```

Comment:
On the yealink configuration template(V80_82) for parameter account.X.dtmf.info_type, values are:It configures the DTMF info type for account X.
1-DTMF-Relay
2-DTMF
3-Telephone-Event